Marlin/language_de.h

```diff
@@ -35,53 +35,53 @@
 #define MSG_AUTO_HOME                       "Home" // G28
 #define MSG_LEVEL_BED_HOMING                "Homing XYZ"
 #define MSG_LEVEL_BED_WAITING               "Klick für Start"
 #define MSG_LEVEL_BED_DONE                  "Fertig"
 #define MSG_LEVEL_BED_CANCEL                "Abbruch"
-#define MSG_SET_HOME_OFFSETS                "Setze Home hier"
+#define MSG_SET_HOME_OFFSETS                "Setze H.-Offsets"
 #define MSG_HOME_OFFSETS_APPLIED            "Offsets aktiv"
-#define MSG_SET_ORIGIN                      "Setze Null hier" //"G92 X0 Y0 Z0" commented out in ultralcd.cpp
+#define MSG_SET_ORIGIN                      "Setze Nullpunkt" //"G92 X0 Y0 Z0" commented out in ultralcd.cpp
 #define MSG_PREHEAT_PLA                     "Vorwärmen PLA"
 #define MSG_PREHEAT_PLA_N                   "Vorwärmen PLA "
 #define MSG_PREHEAT_PLA_ALL                 "Vorw. PLA Alle"
 #define MSG_PREHEAT_PLA_BEDONLY             "Vorw. PLA Bett"
-#define MSG_PREHEAT_PLA_SETTINGS            "Vorwärm. PLA Ein."
+#define MSG_PREHEAT_PLA_SETTINGS            "Vorw. PLA Einstellungen"
 #define MSG_PREHEAT_ABS                     "Vorwärmen ABS"
 #define MSG_PREHEAT_ABS_N                   "Vorwärmen ABS "
 #define MSG_PREHEAT_ABS_ALL                 "Vorw. ABS Alle"
 #define MSG_PREHEAT_ABS_BEDONLY             "Vorw. ABS Bett"
-#define MSG_PREHEAT_ABS_SETTINGS            "Vorwärm. ABS Ein."
+#define MSG_PREHEAT_ABS_SETTINGS            "Vorw. ABS Einstellungen"
 #define MSG_COOLDOWN                        "Abkühlen"
-#define MSG_SWITCH_PS_ON                    "Netzteil Ein"
-#define MSG_SWITCH_PS_OFF                   "Netzteil Aus"
+#define MSG_SWITCH_PS_ON                    "Netzteil ein"
+#define MSG_SWITCH_PS_OFF                   "Netzteil aus"
 #define MSG_EXTRUDE                         "Extrudieren"
 #define MSG_RETRACT                         "Retract"
 #define MSG_MOVE_AXIS                       "Bewegen"
-#define MSG_LEVEL_BED                       "Bett Korrektur"
+#define MSG_LEVEL_BED                       "Bett vermessen"
 #define MSG_MOVE_X                          "X"
 #define MSG_MOVE_Y                          "Y"
 #define MSG_MOVE_Z                          "Z"
 #define MSG_MOVE_E                          "E"
 #define MSG_MOVE_01MM                       " 0.1 mm"
 #define MSG_MOVE_1MM                        " 1.0 mm"
 #define MSG_MOVE_10MM                       "10.0 mm"
-#define MSG_SPEED                           "Geschw."
+#define MSG_SPEED                           "Geschwindigkeit"
 #define MSG_BED_Z                           "Bett Z"
 #define MSG_NOZZLE                          "Düse"
 #define MSG_BED                             "Bett"
-#define MSG_FAN_SPEED                       "Lüftergeschw."
-#define MSG_FLOW                            "Fluss"
+#define MSG_FAN_SPEED                       "Lüftergeschwindigkeit"
+#define MSG_FLOW                            "Durchfluss"
 #define MSG_CONTROL                         "Einstellungen"
```

Comment:
MSG_SPEED eigendlich feedrate_multiplier, 3 Stellen.
MSG_FAN_SPEED in %, 3 Stellen.
MSG_FLOW eigentlich extruder_multiplier, 3 Stellen.